content/docs/5.1/deploy/install/test.md

```diff
@@ -206,7 +206,7 @@ To deploy a TiKV cluster with multiple nodes for test, take the following steps:
                     --log-file=tikv3.log
     ```
 
-You can use the [pd-ctl](https://github.com/pingcap/pd/tree/master/tools/pd-ctl) tool to verify whether PD and TiKV are successfully deployed:
+Use the [pd-ctl](https://github.com/pingcap/pd/tree/master/tools/pd-ctl) tool to verify whether PD and TiKV are successfully deployed:
```

Comment:
To me, there are some slight differences between "you can do" and the imperative "do". In this context, "verify whether PD and TiKV are successfully deployed" looks like a piece of supplementary information in addition to the steps above. It is optional. IMO, "you can use" may look better.

Reply:
Good point. Agree.